shared/lib/constants.py

```diff
@@ -152,198 +154,177 @@
-            "decreasing",
-            "shr(ank|ink|unk)(ing)?",
-            "reduce(d|s)?",
-            "reduc(ing|tion)",
-            "decline(d|s)?",
-            "declining",
-            "plummet(ed|ing|s)?",
-            "fall(en|s)?",
-            "drop(ped|s)?",
-            "loss(es)?",
-        ],
-        "Change": ["change(s|d|) (over|with) time"]
-    },
-    "Superlative": {
-        "Big": ["big", "bigger", "biggest"
-               ],
-        "Small": [
-            "small",
-            "bigger",
-            "smallest"
-        ],
-        "Rich": [
-            "rich",
-            "richer",
-            "richest"
-        ],
-        "Poor": [
-            "poor",
-            "poorer",
-            "poorest"
-        ],
-        "List": ["list( of|)"]
-    },
-    "Overview": ["tell me (more )?about",],
-    # Hint for potential reference to answer-places, to be used
-    # together with ContainedInPlace.
-    "AnswerPlacesReference":
-        ["these",
-         "those"],
-    "PerCapita": [
-        "fraction",
-        "percent",
-        "percentage",
-        "per capita",
-        "percapita",
-        "per person",
-        # remove "rate" if is not followed by certain words (used as one metric)
-        "\brate\b(?!\s*(birth|change|death|exchange|fertility|literacy|mortality|participation|unemployment|withdrawal)\s+rate)",
-        "rates"
-    ],
-    "Temporal": [
-        # Day of week
-        "monday",
-        "tuesday",
-        "wednesday",
-        "thursday",
-        "friday",
-        "saturday",
-        "sunday",
-        # Month of year
-        "january",
-        "february",
-        "march",
-        "april",
-        "may",
-        "june",
-        "july",
-        "august",
-        "september",
-        "october",
-        "november",
-        "december",
-        # Season
-        "spring",
-        "summer",
-        "winter",
-        "autumn",
-        # Others
-        "today",
-        "yesterday",
-        "tomorrow",
-    ]
-}
+        "Temporal": [
+            # Day of week
+            "monday",
+            "tuesday",
+            "wednesday",
+            "thursday",
+            "friday",
+            "saturday",
+            "sunday",
+            # Month of year
+            "january",
+            "february",
+            "march",
+            "april",
+            "may",
+            "june",
+            "july",
+            "august",
+            "september",
+            "october",
+            "november",
+            "december",
+            # Season
+            "spring",
+            "summer",
+            "winter",
+            "autumn",
+            # Others
+            "today",
+            "yesterday",
+            "tomorrow",
+        ]
+    }
 
-# We do not want to strip words from events / superlatives / temporal
+# We do not want to strip words from events / superlatives / temporal / percapita
 # since we want those to match SVs too!
```

Comment:
Have a comment so we will remember why we retain PerCapita words in stop-words in main DC, perhaps an example?

Reply:
done